src/com/twu/biblioteca/model/BookCollection.java

```diff
@@ -0,0 +1,39 @@
+package com.twu.biblioteca.model;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.UUID;
+
+public class BookCollection {
+
+    private HashMap<UUID, Book> collection;
+
+    public BookCollection(ArrayList<Book> bookList) {
+        this.collection = new HashMap<UUID, Book>();
+        for (Book book : bookList) {
+            this.collection.put(book.getId(), book);
+        }
+    }
+
+    public void list() {
+        String format = "%-5s%25s%25s%25s%n";
```

Comment:
Maybe it should be interesting to add this variable and `header` to a `config` object